content/en/users/cloud-compute/openstack/_index.md

```diff
@@ -29,14 +29,39 @@ of this guide will focus on CLI/API access.
 
 ## Installation
 
-TBC
+The OpenStack client is a command-line client for OpenStack that brings the
+command set for Compute, Identity, Image, Object Storage and Block Storage
+APIs together in a single shell with a uniform command structure.
+
+> If installing on Windows, as there are non-pure Python packages needed for
+> installation, please make sure [Microsoft C++ Build Tools](https://visualstudio.microsoft.com/visual-cpp-build-tools/)
+> are installed, with the following options selected:
+>
+> - C++ CMake tools for Windows
+> - C++ ATL for latest v142 build tools (x86 & x64)
+> - Testing tools core features - Build Tools
+> - Windows 10 SDK (`<latest`>)
+>
+> In case you prefer to use non-Microsoft alternatives for building non-pure
+> packages, please see [here](https://wiki.python.org/moin/WindowsCompilers).
+
+Installation of the OpenStack client can be done using:
 
 ```shell
+pip install --upgrade setuptools
 pip install requests
 pip install openstackclient
 ```
 
-- `jq` to easily manage JSON output
+To be able to use sed-like commands to map, filter, slice, and transform
+structured data (JSON), install `jq`:
+
+```shell
+pip install jq
```

Comment:
Is this a python tool? you need to install the binary before (e.g. apt-get install jq)

Reply:
yes, that is a python tool. however, it is not a pure Python package, so if you try to install it using pip it will fail to compile the non-Python parts. at least on my Windows machine. thus the authors also offer it in prebuilt form for installation on windows.

Reply:
and the reference to it is already on the page. I just explained what it actually is and who might need it.

Reply:
So AFAICS it's python binding for jq, not the program itself.
More generally I'm not sure that everyone will want to install software via pip if there is another easier/cleaner/prefered option, like on linux people will probably prefer to use the OS package management (to keep it cleanly installed/updated/removed), and on Mac I've always been using homebrew for this.

I would prefer mentioning https://stedolan.github.io/jq/download/.

Reply:
done

Reply:
I am still unsure that `pip install jq` should be mentioned/recommended.
It's python bindings not the jq binary, it's meant to interact with jq using python (see https://github.com/mwilliamson/jq.py), not much to do with using the jq CLI.
I would just fully remove the pip install jq stuff and only keep the link to the jq binary homepage.

Reply:
it is currently mentioned in the docs. actually, it's the only way mentioned, without any explanation why one would need it. but ok, removing the pip installation.

Reply:
It's not really what I see on https://docs.egi.eu/users/cloud-compute/openstack/#installation.
I see that it's mentioned that it can be used to manage json output, and not mention of installing it via pip, it's one of your commit that introduced this.
![Screenshot 2021-06-30 at 09 14 53](https://user-images.githubusercontent.com/451417/123917981-aeb56a00-d983-11eb-9eb7-4e49f6f9338a.png)

The fact is that `jq` is never used after in the examples, so it feels a bit weird to see it mentioned without any example how it could be used/where it's useful.
Either we should provide some real example on how to use it to parse/format/filter some json output or maybe just removing it from there.

Reply:
See https://github.com/EGI-Federation/documentation/search?q=jq

Reply:
(Now I think that removing it from there is probably the cleanest thing to do)

Reply:
removed